apps/Launcher/images/star.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Yes, I used icons from a set in the public domain: http://publicicons.org/ but there are none that looks good for saving... We can change when Nicolas has time to produce something better :-)